clients/src/main/java/org/apache/kafka/clients/MetadataUpdater.java

```diff
@@ -42,6 +42,12 @@
      */
     boolean isUpdateDue(long now);
 
+    /**
+     * Returns true if the cluster metadata hasn't refreshed for li.client.cluster.metadata.expire.time.ms
+     * and has tried half of the nodes in the cached metadata node set
```

Comment:
change the comments to match implementation?

Reply:
fixed